bin/install-qa-check.d/05prefix

```diff
@@ -90,6 +104,17 @@ install_qa_check_prefix() {
 				# reasons
 				sed -i -e '1s:^#! \?'"${line[0]}"':#!'"${EPREFIX}"${line[0]}':' "${rf}"
 				continue
+			elif [[ -e ${BPREFIX}${line[0]} ]] ; then
+				# is it unprefixed, but we can just fix it because a
+				# bprefixed variant exists
+				eqawarn "bprefixing shebang of ${fn#${D%/}/}"
+				# statement is made idempotent on purpose, because
+				# symlinks may point to the same target, and hence the
+				# same real file may be sedded multiple times since we
+				# read the shebangs in one go upfront for performance
+				# reasons
+				sed -i -e '1s:^#! \?'"${line[0]}"':#!'"${BPREFIX}"${line[0]}':' "${rf}"
```

Comment:
Can we re-use this sed somehow?  e.g. by setting a local var to either EPREFIX or BPREFIX?  Not sure if it's worth it.